src/js/helpers/searchHelper.js

```diff
@@ -62,6 +62,7 @@ export const fetchPSC = (req) => apiRequest({
 });
 
 export const fetchAwardV2 = (awardId) => apiRequest({
+    isLocal: true,
```

Comment:
Shouldn't this line be removed since it seems like the api is already merged into dev?

Reply:
It's not merged just yet.